var/spack/repos/builtin/packages/camp/package.py

```diff
@@ -106,7 +112,14 @@ def cmake_args(self):
 
         options.append(self.define_from_variant("ENABLE_HIP", "rocm"))
         if spec.satisfies("+rocm"):
-            options.append("-DHIP_ROOT_DIR={0}".format(spec["hip"].prefix))
+            rocm_root = dirname(spec["llvm-amdgpu"].prefix)
+            options.append("-DROCM_PATH={0}".format(rocm_root))
+
+            # there is only one dir like this, but the version component is unknown
+            options.append(
+                "-DHIP_CLANG_INCLUDE_PATH="
```

Comment:
Is this still required?

Reply:
Maybe not. I can test it, but I have no reproducer of the original scenario justifying this, so I can't prove anything.

I am going to build Umpire with rocm from 5.7.1 to 6.3.1 and see if we have any issues.